configure.ac

```diff
@@ -212,6 +212,8 @@ AC_CHECK_TYPES([struct xt_entry_match, struct xt_entry_target],,,[
 #include <linux/netfilter_ipv4/ip_tables.h>
 ])
 
+AC_CHECK_TYPES([struct __kernel_old_timeval, struct __kernel_old_timespec, struct __kernel_timespec, struct __kernel_old_itimerspec, struct __kernel_itimerspec],,,[#include <sys/socket.h>])
```

Comment:
Only a minor thing, but can we split this line?

Otherwise LGTM, thanks!

Reply:
Done